### DISCUSSION TITLE
Bypass open/close

### DISCUSSION THREAD

**Discussion by cheick66** [yozik04/vallox_websocket_api#36] [Q&A]
Hi,
I have a Vallox 270MV connected via LAN/ModBus .
Is it possible to open/close the bypass? I found a lot of registers but can't figure out which to change.

1_BYPASS_LOCKED_CYC | 0 | 2023-03-16 16:45:30
1_BYPASS_LOCKED_EXT | undefined | 2023-03-16 16:45:30
1_BYPASS_MODE | undefined | 2023-03-16 16:45:30
1_CELL_STATE_BYPASS | 0 | 2023-03-16 16:45:30
1_DIG_IN_BYPASS | 0 | 2023-03-16 16:45:30
1_IN_BYPASS | 0 | 2023-03-16 16:45:30
1_IO_BYPASS | 0 | 2023-03-16 16:45:30
1_PARTIAL_BYPASS | 0 | 2023-03-16 16:45:30
1_RELAY_BYPASS_STATE | 0

Can you give me some help?

Best regards
   Christian

**Comment by yozik04** [comment]
I personally only control bypass lock. It prevents bypass opening. Not sure if other registers can be used or not. You can experiment. By setting temperature you can force it to open.
